packages/ui/src/components/Tag/tag.tsx

```diff
@@ -8,7 +8,7 @@ import { colors } from '@open-condo/ui/src/colors'
 const TAG_CLASS_PREFIX = 'condo-tag'
 
 export type TagProps = React.HTMLAttributes<HTMLSpanElement> & {
-    children: string
+    children: React.ReactNode
```

Comment:
Why do you need Node here?

Reply:
I need to mark up text with an icon

Reply:
Yes, the default component has an icon property, but that icon is on the left. But I need to place it on the right. And instead of having to hardcode the position of the icon in the component, it's better to allow any content to be added.

it seems we have already encountered this problem more than once, and changed interfaces